### ISSUE TITLE
Accessibility improvements on Tooltip component

### ISSUE THREAD

**Issue by muratcorlu** [Trendyol/baklava#286]
- [ ] Tooltip related `aria-` attributes should be used in component. [More info](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role)
